index.html

```diff
@@ -0,0 +1,193 @@
+      </header>
+
+      <ul class="work-row">
+        <li class="item">
+          <img class="placeholder" src="./pics/Img-placeholder.png" alt="">
+          <div class="text-card">
+            <h3 class="subtitle">Multi-Post Stories Gain+Glory</h3>
+            <ul class="courses">
+              <li>Ruby on Rails</li>
+              <li>CSS</li>
+              <li>JavaScript</li>
+              <li>HTML</li>
+            </ul>
+          <button type="submit">See Project</button>
+          </div>
+        </li>
+        <li class="item">
+          <img class="placeholder" src="./pics/Img-placeholder.png" alt="">
+          <div class="text-card">
+            <h3 class="subtitle">Multi-Post Stories Gain+Glory</h3>
+            <ul class="courses">
+              <li>Ruby on Rails</li>
+              <li>CSS</li>
+              <li>JavaScript</li>
+              <li>HTML</li>
+            </ul>
+            <button type="submit">See Project</button>
+          </div>
+        </li>
+        <li class="item">
+          <img class="placeholder" src="./pics/Img-placeholder.png" alt="">
+          <div class="text-card">
+            <h3 class="subtitle">Multi-Post Stories Gain+Glory</h3>
+            <ul class="courses">
+              <li>Ruby on Rails</li>
+              <li>CSS</li>
+              <li>JavaScript</li>
+              <li>HTML</li>
+            </ul>
+            <button type="submit">See Project</button>
+          </div>
+      </li>
+        <li class="item">
+          <img class="placeholder" src="./pics/Img-placeholder.png" alt="">
+          <div class="text-card">
+            <h3 class="subtitle">Multi-Post Stories Gain+Glory</h3>
+            <ul class="courses">
+              <li>Ruby on Rails</li>
+              <li>CSS</li>
+              <li>JavaScript</li>
+              <li>HTML</li>
+            </ul>
+            <button type="submit">See Project</button>
+          </div>
+      </li>
+        <li class="item">
+          <img class="placeholder" src="./pics/Img-placeholder.png" alt="">
+          <div class="text-card">
+            <h3 class="subtitle">Multi-Post Stories Gain+Glory</h3>
+            <ul class="courses">
+              <li>Ruby on Rails</li>
+              <li>CSS</li>
+              <li>JavaScript</li>
+              <li>HTML</li>
+            </ul>
+            <button type="submit">See Project</button>
+          </div>
+        </li>
+        <li class="item">
+          <img class="placeholder" src="./pics/Img-placeholder.png" alt="">
+          <div class="text-card">
+            <h3 class="subtitle">Multi-Post Stories Gain+Glory</h3>
+            <ul class="courses">
+              <li>Ruby on Rails</li>
+              <li>CSS</li>
+              <li>JavaScript</li>
+              <li>HTML</li>
+            </ul>
+            <button type="submit">See Project</button>
+        </div>
+        </li>
+      </ul>
+    </section>
+
+    <section id="about-me">
```

Comment:
Kindly also try to fix the background image of the `about-me` section 🤗

Your design:

![image](https://user-images.githubusercontent.com/73714615/187036112-576da8e1-274f-4446-88d5-5999a5aa73e7.png)

Figma design:

![image](https://user-images.githubusercontent.com/73714615/187036126-d63258cf-5141-4c38-ac93-b8ab30c4071b.png)